deployment/volumes/volume-snapshots/README.md

```diff
@@ -9,13 +9,27 @@ Kubernetes announced this feature in a [blog post](https://kubernetes.io/blog/20
 ## Create a Volume Snapshot
 
 There is an example in this folder how you can e.g. create a volume snapshot for
-the persistent volume claim `neo4j-data-claim`:
+the persistent volume claim `neo4j-data-claim` or any other claim.
+
+Before you create the snapshot, it is good practice to look at what snapshots are already done:
+
+```sh
+# list existing snapshots
+kubectl get volumesnapshots -n human-connection
+```
+
+To specify your snapshot, edit the file `snapshot.yaml` or simply edit and use `neo4j-data.yaml` in this folder.
```

Comment:
you don't want to ever edit and apply the `neo4j-data.yaml`! this is for creating new `volumes` not for anything to do with `volumesnapshots`

you would probably get an error that the `neo4j-data-claim` already exists, but still there is no need to ever apply this file, unless you are setting up a new `persistent volume`

Reply:
I have to speak about this to you in person, because I get not safely the point. @mattwr18

Reply:
we can do that, the point is that you should not edit the `neo4-data.yaml` since it is for setting up `PersistentVolumeClaims`/`PersistentVolumes`, not for creating a `VolumeSnapshot`.
Also, it's potentially dangerous.

Reply:
Okay, I get this.
Thanks! @mattwr18